### ISSUE TITLE
turning off "citadel replied to so-and-so" logging doesn't seem to work

### ISSUE THREAD

**Issue by GavJenks** [DevotedMC/Citadel#61]
![image](https://user-images.githubusercontent.com/1256888/64573941-1a979000-d322-11e9-83ba-85fdc6fcc20c.png)

**Comment by GavJenks** [comment]
Citadel 3.11.4

**Comment by ProgrammerDan** [comment]
that's hostile_logging, though, no?

**Comment by GavJenks** [comment]
Sounds like command logging to me from the description:
"command_logging logs the response that Citadel sends to the player;"
"[INFO]: Citadel Sent so-and-so a reply ____"

But Ill try damage logging too

**Comment by ProgrammerDan** [comment]
The way it trends down in the screenshot -- that's someone breaking a
chest, it's a hostile break log.

**Comment by ProgrammerDan** [comment]
Yeah, no, you're right -- that's command logging. Been a long time since I
looked at this code. Bad name for the config line I think, basically that
hooks into the sendAndLog() function, and is of the format "Sent <player>
reply <msg>". Confused me, b/c I forgot about block solidification being
standard now :D.

So, the issue is that setting command_logging: false doesn't do anything?

**Comment by GavJenks** [comment]
I don't even know anymore, I'll retest both and get back to you

**Comment by ProgrammerDan** [comment]
Oy, sorry for the confusion -- thank you!
